docs/project/changelog.md

```diff
@@ -164,7 +164,9 @@ myst:
 
 ### Packages
 
-- New packages: fastparquet {pr}`3590`, cramjam {pr}`3590`.
+- New packages: fastparquet {pr}`3590`, cramjam {pr}`3590`, sourmash {pr}`3635,
+screed {pr}`3635, bitstring {pr}`3635, deprecation {pr}`3635, cachetools
```

Comment:
I think it is okay to add only `sourmash` here, since other packages are pure python packages.

Reply:
Got it! I ended up adding them because when I tried to test locally they didn't seem to be read as dependencies unless listed in the `meta.yaml` file, do you have pointers on how to declare them?

Reply:
You are doing it the right way. We do dependency resolution before downloading a package for in-tree packages, so it has to be explicitly declared in meta.yaml and all dependencies have to be added in-tree (which I want to change in the future https://github.com/pyodide/pyodide/issues/2580.)